packages/core/directives/map.ts

```diff
@@ -430,7 +433,11 @@ export class AgmMap implements OnChanges, OnInit, OnDestroy {
     if (typeof this.latitude !== 'number' || typeof this.longitude !== 'number') {
       return;
     }
-    this._setCenter();
+
+    // apply changes to map
+    if (this.latitude !== this._mapLatitude || this.longitude !== this._mapLongitude) {
```

Comment:
Is this accurate? For example, 0.1 + 0.2 in JavaScript isn't even close to 0.3.  If you use LatLng you can use its equals operator.

Reply:
Yes, this is accurate. What we're checking here is, if we already set the values to the latitude and longitude, in which case, we shouldn't do it again.
Basically 
```
a = b;
if (a !== b)
  a = b
```

Btw, 0.1+0.2 is very close to 0.3, it is equal to 0.3 to the 16th decimal.